core/settings/qPrefDisplay.h

```diff
@@ -25,6 +25,7 @@ class qPrefDisplay : public QObject {
 	Q_PROPERTY(QByteArray windowState READ windowState WRITE set_windowState NOTIFY windowStateChanged)
 	Q_PROPERTY(int lastState READ lastState WRITE set_lastState NOTIFY lastStateChanged)
 	Q_PROPERTY(bool singleColumnPortrait READ singleColumnPortrait WRITE set_singleColumnPortrait NOTIFY singleColumnPortraitChanged)
+	Q_PROPERTY(bool standard_level_grid READ standard_level_grid WRITE set_standard_level_grid NOTIFY standard_level_gridChanged)
```

Comment:
of course the different name will affect almost everything you did. sorry.

Reply:
That's fine. I was expecting a battle over variable naming. :)